eks/examples/cnpack/aws-pca.tf

```diff
@@ -9,6 +9,7 @@ AWS Private Cert Authority Config
 resource "aws_acmpca_certificate_authority" "cnpack-pca" {
   count = var.pca_enabled ? 1 : 0
   type  = "ROOT"
+  usage_mode = var.pca_short_lived ? "SHORT_LIVED_CERTIFICATE" : "GENERAL_PURPOSE"
```

Comment:
Seems to agree with https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acmpca_certificate_authority#usage_mode